client/src/components/home/footer.tsx

```diff
@@ -52,9 +52,9 @@ const Footer = () => (
             <LinkedinIcon className="min-w-4 h-4 w-4" />
             <span className="sr-only">LinkedIn</span>
           </Link>
-          <Link href="#">
-            <MailIcon className="min-w-5 h-4 w-5 hover:opacity-90" />
-          </Link>
+          {/* <Link href="#">
```

Comment:
We do have the Mail link on the designs

Reply:
I've temporarily removed it because the link is empty but I forgot to add a task on Jira. I will do so now.

Once we get the email address, I will add back this link. What do you think?

Reply:
Emilie answered very quickly with the email address to use so I've updated this PR.